posconvert.module

```diff
@@ -236,7 +268,12 @@ function posconvert_form($form, &$form_state) {
         $lines = $file_row_count;
       }
 
-      if ($cols < 2 || count($check_rows) <= 0) {
+      if ($lines > 19500) {
```

Comment:
I've created a function that will allow these values (line and size limit) to be accessible to the entire module where it is required.